.github/workflows/tests.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Cypress Tests
+on: [push]
```

Comment:
Let's run these on 
- pull requests
- Merges to master (this is the dev deployment) or env/uat (this is the uat deployment)
- Tags starting with `v` (this is the prod deployment)
Check out the other files in `.github/workflow`

Reply:
fixed in latest commit